### PR TITLE
Fix commander

### PR DESCRIPTION
**Pull request by HsuTing**
- `arguments` in `outputHelp` should be a function
  - [api docs](http://tj.github.io/commander.js/#Command.prototype.outputHelp)
- `command.parse` can return `argv`, `rawArgv` and `options`.
  - [api docs](http://tj.github.io/commander.js/#Command.prototype.parse)

### CODE REVIEW

File: definitions/npm/commander_v2.x.x/flow_v0.28.x-/commander_v2.x.x.js [hunk 1]
```diff
@@ -159,7 +159,12 @@ declare module "commander" {
      * @return {Command} for chaining
      * @api public
      */
-    parse(argv: Array<string>): this;
+    parse(argv: Array<string>): {
+      ...this,
+      args: Array<string>,
+      rawArgs: Array<string>,
+      [string]: any,
+    };
```
Comment (HsuTing) [inline definitions/npm/commander_v2.x.x/flow_v0.28.x-/commander_v2.x.x.js]: I am not sure that this is the best way to fix it.
If you have a good idea to modify this, tell me please.

File: definitions/npm/commander_v2.x.x/flow_v0.28.x-/commander_v2.x.x.js [hunk 2]
```diff
@@ -243,7 +248,7 @@ declare module "commander" {
      *
      * @api public
      */
-    outputHelp(): void;
+    outputHelp((defaultHelp: string) => string): void;
```
Comment (villesau) [inline definitions/npm/commander_v2.x.x/flow_v0.28.x-/commander_v2.x.x.js]: based on docs, this is optional, you can use e.g `outputHelp(cb?: ?(defaultHelp: string) => string): void;`

File: definitions/npm/commander_v2.x.x/flow_v0.28.x-/commander_v2.x.x.js (hunk 1 above)
Comment (villesau) [reply definitions/npm/commander_v2.x.x/flow_v0.28.x-/commander_v2.x.x.js]: Other way could be `this & { args: Array<string>, ... }`. I'm not sure which is better.

File: definitions/npm/commander_v2.x.x/flow_v0.28.x-/commander_v2.x.x.js (hunk 2 above)
Comment (HsuTing) [reply definitions/npm/commander_v2.x.x/flow_v0.28.x-/commander_v2.x.x.js]: OK, I will fix this.

File: definitions/npm/commander_v2.x.x/flow_v0.28.x-/commander_v2.x.x.js (hunk 1 above)
Comment (HsuTing) [reply definitions/npm/commander_v2.x.x/flow_v0.28.x-/commander_v2.x.x.js]: I like your answer. `this & { args: Array<string>, ... }` make this more understandable.
I will change.

**Comment by villesau** [comment]
Great, thanks!

**Pull request merged**